iotedgehubdev/edgemanager.py

```diff
@@ -340,6 +380,10 @@ def _getModuleReqUri(self, name):
         return "https://{0}/devices/{1}/modules/{2}?api-version=2018-06-30".format(
             self._hostname, self._device_id, name)
 
+    def _get_update_twin_uri(self, name):
+        return "https://{0}/twins/{1}/modules/{2}?api-version=2018-06-30".format(
```

Comment:
I see we also use REST API in other code area. Would it be better to use Python SDK directly?

Reply:
python sdk is not well implemented yet. (a wrapper out of c). So since it's just use simple rest call